combustion.gemspec

```diff
@@ -10,11 +10,9 @@ Gem::Specification.new do |s|
   s.description = "Test your Rails Engines without needing a full Rails app"
   s.license     = "MIT"
 
-  s.files         = `git ls-files`.split("\n")
-  s.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
-  s.executables   = `git ls-files -- exe/*`.split("\n").map do |file|
-    File.basename(file)
-  end
+  s.files         = Dir["{exe,lib,templates}/**/*", "HISTORY", "LICENCE", "README.md"]
```

Comment:
Do we want to make sure tests can be run in the packaged file? If so, do we want a few more files?

* .rspec
* Appraisals
* Gemfile
* And maybe Rakefile?

Reply:
I only mentioned these files because the general philosophy is to only ship the files that are needed by the end-users and that to never ship the tests. This works well with downstreams because they take the entire source from g/h tarballs :)

But this is more of a personal preference here! Do you want me to include these files as well? I'd be happy to if you want?
(but I don't think there's need for it to but your call entirely! 😄)

Reply:
Maybe the files I listed should be included in the `test_files` list? That'd make more sense - they're certainly not required for the operation of the gem itself, but they're pretty essential for running the tests.

Reply:
Definitely! Added in the latest push! 😄